standalone/src/models/custom-application/custom-application-menu-link/types.ts

```diff
@@ -1,28 +1,9 @@
 import type { TBuilder } from '@/core';
-import type {
-  TMcSettingsCustomApplicationMenuLink,
-  TMcSettingsCustomApplicationMenuLinkDraftDataInput,
-} from '@/graphql-types';
+import type { TMcSettingsCustomApplicationMenuLink } from '@/graphql-types';
 
-export type TCustomApplicationMenuLinkGraphql = Omit<
-  TMcSettingsCustomApplicationMenuLink,
-  'submenuLinks'
->;
+export type TCustomApplicationMenuLinkGraphql =
+  TMcSettingsCustomApplicationMenuLink;
 
-export type TCustomApplicationMenuLink = Omit<
-  TCustomApplicationMenuLinkGraphql,
-  '__typename'
->;
-
-export type TCustomApplicationMenuLinkDraft =
```

Comment:
Perhaps that was explained before but, why did we remove the draft types from this file?

Usually we keep all the types in the same file and share them for both the read and the write models of an entity.

Reply:
fixed in https://github.com/commercetools/test-data/pull/827/commits/0560e3787880fb668a77fcea122e3cac3b6e5826